tests/test_isrTaskLSST.py

```diff
@@ -476,6 +476,79 @@ def test_isrFlat(self):
         ratio = result2.exposure.image.array / result.exposure.image.array
         self.assertFloatsAlmostEqual(ratio, flat_nodefects.image.array, atol=1e-4)
 
+    def test_isrBrighterFatter(self):
+        """Test processing of a flat frame."""
+        mock_config = self.get_mock_config_no_signal()
+        mock_config.doAddDark = True
+        mock_config.doAddFlat = True
+        # The doAddSky option adds the equivalent of flat-field flux.
+        mock_config.doAddSky = True
+        mock_config.doAddSiliconPhysics = True
+        mock_config.bfStrength = 1.0
+        mock_config.diffusionStrength = 0.0
+        mock_config.nRecalc = 10_000
+
+        mock = isrMockLSST.IsrMockLSST(config=mock_config)
+        input_exp = mock.run()
+
+        isr_config = self.get_isr_config_electronic_corrections()
+        isr_config.doBias = True
+        isr_config.doDark = True
+        isr_config.doFlat = True
+        isr_config.doDefect = True
+        isr_config.doBrighterFatter = False
```

Comment:
We are not running the bf correction here, which seems incorrect.

Reply:
Oops remnant from a test. Fixed. 
Made sure the same test cases passes ✅